uCNC/src/modules/softspi.c

```diff
@@ -190,6 +186,23 @@ void softspi_bulk_xmit(softspi_port_t *port, const uint8_t *out, uint8_t *in, ui
 	}
 }
 
+bool softspi_isbusy(softspi_port_t *port)
+{
+	if (!port)
+	{
+#ifdef MCU_HAS_SPI
+		port->spiconfig.locked = mcu_spi_port.isbusy;
```

Comment:
Looks like this is assigning to a null pointer

Reply:
Thanks